js/main.js

```diff
@@ -20,4 +20,88 @@ $entryForm.addEventListener('submit', function (event) {
   data.entries.unshift(singleData);
   $imagePlaceholder.setAttribute('src', entryImagePlaceholder);
   $entryForm.reset();
+
+  // const $renderEntry = renderEntry(singleData);
+  entriesList.prepend(renderEntry(singleData));
+  viewSwap('entries');
+  data.view = 'entries';
```

Comment:
You shouldn't be setting `data.view` here, this should only get set inside of `viewSwap`